canopas/vue-file-upload
# Manage multiple file upload

> PR Description:
- Made a component with considering custom design(by user) for managing multiple file upload with preview
- Implement call-back functionality for showing the process/uploading view of files
- Refactor single file upload management component
- Exported single file upload and multiple file upload management components
- Update `README.md` file for considering whole library functionality step by step
- Tested library by linking npm in the local

> Test Steps:
- You can check the `README.md` on https://github.com/canopas/file-upload-web-frontend/tree/upload-multiple-files
- For testing the component, checkout this branch and run the server using `npm run dev`
- You can comment and uncomment the code on the `IndexView.vue` file for testing both modules
